getting-started-5.html

```diff
@@ -499,13 +507,13 @@
 
 </li>
 
-<li><p class="first">Click on the OMERO.insight link in the pop-up window.</p>
+<li><p class="first">Click on the <span class="bold">OMERO.insight</span> link in the pop-up window.</p>
 
 <div class="line-block">
 <div class="line"><br /></div> <!-- end #line -->
 </div> <!-- end #line-block -->
 
-<img alt="../images/gettingStarted5WebStartPopup.jpg" class="align-center" src="images/gettingStarted5WebStartPopup.jpg" style="width: 60%;" />
+<img alt="../images/gettingStarted5WebstartPopUp.jpg" class="align-center" src="images/gettingStarted5WebstartPopUp.jpg" style="width: 60%;" />
```

Comment:
@gusferguson this line needs changing back to /images/gettingStarted5WebStartPopup.jpg